docs/images/build-process.dot

```diff
@@ -0,0 +1,45 @@
+digraph G {
+    rankdir=LR;
+
+    // Subgraph for packages
+    subgraph cluster_packages {
+        label="Packages";
+        node [shape=box, height=0.5, width=1.5, style=filled, fillcolor="lightyellow", color="black"]; // Packages with light yellow background and black border
+
+        // Define packages
+        rngd [label="rngd"];
+        sysctl [label="sysctl"];
```

Comment:
the sysctl package doesn't exist (maybe it was removed in the past?)....

Reply:
It does, but it is not in `pkg/` but rather upstream, see [images/rootfs.yml.in](https://github.com/lf-edge/eve/blob/master/images/rootfs.yml.in#L21-L27):

```yaml
  - name: sysctl
    image: linuxkit/sysctl:c6f23919b8610c7645a89a89f863c6209bc84bee
    binds:
      - /etc/sysctl.d:/etc/sysctl.d
    capabilities:
      - CAP_SYS_ADMIN
      - CAP_NET_ADMIN
```

Reply:
oh, got it, thanks for pointing out...